### ISSUE TITLE
Focus styles in header not obvious

### ISSUE THREAD

**Issue by anevins12** [WordPress/twentytwenty#122]
I can see that the header interactive elements have had their default focus indicator removed. That's fine as long as a more obvious indicator is provided. 

I've taken a screenshot to illustrate the lack of clarity in the currently implemented focus indicator.. Can you tell element I have focused on? 
![16baa6a0-5c87-4625-9859-1a6c2b2bb769](https://user-images.githubusercontent.com/2318561/64494461-208b6380-d285-11e9-8461-0bcf4baef502.png)

Let's continue to use the default browser focus indicator. If we want to change that then it should be a wider design strategy that we can make consistent, predictable and obvious to users.

**Comment by andersnoren** [comment]
Agreed. We're considering using the following focus/hover state for the toggles in the header:

![Single → Header Toggle Focus State → Desktop](https://user-images.githubusercontent.com/8181091/64494508-25a0e080-d28e-11e9-93ac-775a1f7e22f2.jpg)

The pattern of displaying an underlying circle as a focus/hover state could be reused for the sub navigation toggles in the expanding menu on desktop and the mobile menu as well.

**Comment by carolinan** [comment]
I believe this has been fixed with the addition of the text below the icons.
The text is underlined when focused.

**Comment by andersnoren** [comment]
The hover effect in the mockup didn't work well with the text labels (#93), so we decided against adding it.
